RC: Active-Active with Search (#139)
redis/docs: content/operate/rc/databases/create-database/create-active-active-database.md

Throughput requirements grow dramatically as regions increase.  As a result, consider your requirements carefully.

> For Search and query databases, the estimated throughput from the [Sizing calculator](https://redis.io/redisearch-sizing-calculator/) is the total amount of additional throughput you need. When setting throughput for your Active-Active database, add this throughput divided across each region depending on your needs for each region.

Reviewer: ```suggestion
For Search and query databases, the estimated throughput from the [Sizing calculator](https://redis.io/redisearch-sizing-calculator/) is the total amount of throughput you need. When setting throughput for your Active-Active database use the total amount and divided it across each region depending on your query and write needs for each region.
```
It's not an additional throughput, since it's one estimation based on the amount of shards you need. So if you have 4 shards the TP is 4x25000 = 100000. So it's covers overall TP need

> 

Reviewer: Should we consider a simple example here? Where the output from the calculator, for example 50000 is divided in 20000 writes(updates) and 30000 reads(queries)

Author: I added an example and clarified the throughput recommendations! Let me know what you think.
